include/tera/mutation.h

```diff
@@ -5,25 +5,83 @@
 #ifndef  TERA_MUTATION_H_
 #define  TERA_MUTATION_H_
 
-#include <stdint.h>
-#include <list>
-#include <map>
-#include <set>
 #include <string>
-#include <vector>
+
+#include "error_code.h"
 
 #pragma GCC visibility push(default)
 namespace tera {
 
-class RowLock {
-};
-
-class Transaction;
 class Table;
-/// 修改操作
+class Transaction;
+class RowLock;
+
+// Instantiated through tera::Table::NewRowMutation.
+// RowMutation is a list of operations(update/delete) named "mutation". All
+// mutations will be committed to server by tera::Table::Put atomically.
 class RowMutation {
-/// rowkey的限制：大小 [0, 64KB)，任意二进制串
 public:
+    // Return current row key.
+    virtual const std::string& RowKey() = 0;
+
+    // Put "value" in key-value mode.
+    // "ttl"(time-to-live) is optional, "value" will expire after "ttl"
+    // second. If ttl <= 0, "value" never expire.
+    virtual void Put(const std::string& value, int32_t ttl = -1) = 0;
+
+    // Put a cell in table mode.
+    // "timestamp"(us) is optional, current timestamp by default.
```

Comment:
https://www.google.com/search?newwindow=1&q=if+not+specified++is+used&oq=if+not+specified++is+used&gs_l=serp.3..0i8i10i30k1.33231.34576.0.36507.9.9.0.0.0.0.153.736.4j3.7.0....0...1c.1.64.serp..2.4.398...30i10k1.3gK4gGVuOcc